.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: DR2 Tag and pre deploy
+on:
+  push:
+    branches:
+      - main
+permissions:
+  id-token: write
+  contents: write
+jobs:
+  pre-deploy:
+    uses: nationalarchives/dr2-github-actions/.github/workflows/lambda_build.yml@main
+    with:
+      repo-name: dr2-entity-event-generator
+      build-command: sbt assembly
```

Comment:
There's a required argument `artifact-name` which will be `entityEventGenerator` in this case.